### DISCUSSION TITLE
Multiple Baselines Set error - Jamf Pro remediation

### DISCUSSION THREAD

**Discussion by n-schoofs** [usnistgov/macos_security#309] [Q&A] [answered]
Hi there

Tinkering resulted in some of our Macs that have multiple baselines set in` /Library/Preferences`. Any recommendations, best practice, examples or maybe compliance script functionality that I can use to delete the baselines?

I'm thinking about creating a Smart Group checking for "Multiple Baselines Set" in the Compliance Version Extension Attribute in Jamf Pro. Then a remediation script in a Policy to delete the baselines found in `/Library/Preferences`, followed by a `--check` from the compliance script and an Inventory Update configured in the Policy.

WDYT?

**Comment by brodjieski** [comment, marked as answer]
Running a policy to delete all of your compliance .plist files from other baselines is the way to go. That will clear out any old baseline results or other information that is likely not needed anymore. Then running the `--check` on the baseline you want to maintain will then only have the results from that baseline.

If you still have the compliance scripts from the other baselines scoped and available to the systems, you can run it with `--reset` which will clear out the .plist as well.
